1.11/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gunicorn==19.9.0
-django==1.11.17
+django==1.11.18
 psycopg2==2.7.6.1 --no-binary psycopg2
 gevent==1.3.7
```

Comment:
We can bump this to `1.4.0`.

Reply:
Thanks for catching this, completely forgot to check for package updates in `gevent`. Will update it in all supported Django versions as part of this PR.